### PR TITLE
D-26871 Session cookie name change for nginx not to clash with JSESSIONID

### PR DESCRIPTION
**Pull request by PatrickDei**

### CODE REVIEW

File: xl-op/digitalai/dai-deploy/kubernetes-generic/dai-deploy_cr_default.yaml.tmpl [hunk 1]
```diff
@@ -113,7 +113,7 @@ spec:
       nginx.ingress.kubernetes.io/proxy-read-timeout: "60"
       nginx.ingress.kubernetes.io/proxy-send-timeout: "60"
       nginx.ingress.kubernetes.io/rewrite-target: /
-      nginx.ingress.kubernetes.io/session-cookie-name: JSESSIONID
+      nginx.ingress.kubernetes.io/session-cookie-name: ROUTE
```
Comment (vpugar-digital) [inline xl-op/digitalai/dai-deploy/kubernetes-generic/dai-deploy_cr_default.yaml.tmpl:116]: Please check if that is happening for the haproxy, I think we need there change that config and for external too, check lines 129 and 141. Do we need same change on the openshift too?

File: xl-op/digitalai/dai-release/kubernetes-generic/dai-release_cr_default.yaml.tmpl [hunk 2]
```diff
@@ -101,7 +101,7 @@ spec:
       nginx.ingress.kubernetes.io/proxy-read-timeout: "60"
       nginx.ingress.kubernetes.io/proxy-send-timeout: "60"
       nginx.ingress.kubernetes.io/rewrite-target: /
-      nginx.ingress.kubernetes.io/session-cookie-name: JSESSIONID
+      nginx.ingress.kubernetes.io/session-cookie-name: ROUTE
```
Comment (vpugar-digital) [inline xl-op/digitalai/dai-release/kubernetes-generic/dai-release_cr_default.yaml.tmpl:104]: Same comment as for deploy:
Please check if that is happening for the haproxy, I think we need there change that config and for external too, check lines 129 and 141. Do we need the same change on the openshift too?

**Comment by vpugar-digital** [comment]
Create PR for the 23.3.x branch too

File: xl-op/digitalai/dai-deploy/kubernetes-generic/dai-deploy_cr_default.yaml.tmpl (hunk 1 above)
Comment (PatrickDei) [reply xl-op/digitalai/dai-deploy/kubernetes-generic/dai-deploy_cr_default.yaml.tmpl:116]: I tried haproxy and it worked as configured already, without changes. openshift was confirmed to be working when the bug was reported by Inba if I recall correctly, does that cover all the bases?

Comment (PatrickDei) [reply xl-op/digitalai/dai-deploy/kubernetes-generic/dai-deploy_cr_default.yaml.tmpl:116]: haproxy and openshift work as expected with the current configuration :)

**Review by vpugar-digital — APPROVED** [review]

**Pull request merged**